tests/unit_tests/test_libres_facade.py

```diff
@@ -281,40 +285,42 @@ def test_summary_collector(monkeypatch, facade):
         )
 
 
-def test_misfit_collector(facade):
+def test_misfit_collector(snake_oil_case_storage):
+    facade = LibresFacade(snake_oil_case_storage)
     data = facade.load_all_misfit_data("default_0")
 
-    assert pytest.approx(data["MISFIT:FOPR"][0]) == 738.735586
-    assert pytest.approx(data["MISFIT:FOPR"][24]) == 1260.086789
+    assert pytest.approx(data["MISFIT:FOPR"][0]) == 1966.3584925069108
+    assert pytest.approx(data["MISFIT:FOPR"][4]) == 758.1662249270752
 
-    assert pytest.approx(data["MISFIT:TOTAL"][0]) == 767.008457
-    assert pytest.approx(data["MISFIT:TOTAL"][24]) == 1359.172803
+    assert pytest.approx(data["MISFIT:TOTAL"][0]) == 2024.461010928874
+    assert pytest.approx(data["MISFIT:TOTAL"][4]) == 831.520534770889
 
     # pylint: disable=pointless-statement
     # realization 20:
-    data.loc[20]
+    data.loc[2]
```

Comment:
what does this do?

Reply:
:shrug: `# pylint: disable=pointless-statement` ? Who knows what the intent was, I have removed it.